src/utils/interfaces.ts

```diff
@@ -185,3 +185,44 @@ export interface DocumentHit extends Document {
    */
   _score: number;
 }
+
+export interface MappingsProperties {
+  /**
+   * Properties types definition
+   *
+   * @see https://docs.kuzzle.io/core/2/guides/essentials/database-mappings/#properties-types-definition
+   */
+  properties?: MappingsProperties,
+  /**
+   * Dynamic mapping policy
+   *
+   * @see https://docs.kuzzle.io/core/2/guides/essentials/database-mappings/#dynamic-mapping-policy
+   */
+  dynamic?: 'true' | 'false' | 'strict'
```

Comment:
As per https://github.com/kuzzleio/kuzzle/pull/1745, shouldn't we also accept boolean values?